3_Implementation/documentation/html/search/typedefs_0.js

```diff
@@ -0,0 +1,15 @@
+var searchData=
+[
+  ['unity_5fdouble_895',['UNITY_DOUBLE',['../unity__internals_8h.html#ac3bdb18f38e23a4343af34a7f4e0d4a0',1,'unity_internals.h']]],
+  ['unity_5ffloat_896',['UNITY_FLOAT',['../unity__internals_8h.html#ac4a36d11e8a7b250f2413ebb22e3092c',1,'unity_internals.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unity_5ffloat_896",['UNITY_FLOAT',['../unity__internals_8h.html#ac4a36d11e8a7b250f2413ebb22e3092c',1,'unity_internals.h']]],
```